### ISSUE TITLE
Issue with external links

### ISSUE THREAD

**Issue by dolbex** [michalzaq12/electron-nuxt#718]
In the template there is an error: `Cannot read property 'shell' of undefined ` when trying to execute external links with `remote.shell.openExternal`. Any idea why we can't import remote from Electron?

**Comment by dolbex** [comment]
The magic of opening a ticket...

It appears this should be accessed this way:

```
import { shell } from "electron";
```
...

```
shell.openExternal('https://google.com');
```

perhaps the template should be updated?

**Comment by michalzaq12** [comment]
Fixed in 1.8.1
